lib/core/include/irods/msParam.h

```diff
@@ -94,8 +94,6 @@ typedef struct ValidKeyWd {
 
 #define RESC_NAME_FLAG          0x1
 #define DEST_RESC_NAME_FLAG     0x2
-// BACKUP_RESC_NAME has been deprecated.
-#define BACKUP_RESC_NAME_FLAG   0x4
```

Comment:
Hmm, wondering if we should leave this in but commented out.

Reply:
Did we want to leave this here and leave a comment after all?

Reply:
Yes. Having it there, but commented out keeps us from reusing old values.

Reply:
Done.